src/Sdk/Cerbos/Sdk/Cloud/V1/Store/FileFilter.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace Cerbos.Sdk.Cloud.V1.Store
+{
+    public sealed class FileFilter
+    {
+        private StringMatch Path { get; set; }
+
+        private FileFilter() { }
+
+        public static FileFilter NewInstance()
+        {
+            return new FileFilter();
+        }
+
+        public FileFilter WithPath(StringMatch path)
```

Comment:
This would be confusing to users because `WithPath` sounds like we just want a single path. In reality, there are 3 different filters: `exact`, `like` and `in`. We should make it easier to construct either one of those with static constructors like `WithPathEqual`, `WithPathLike` and `WithPathIn`.